connect/src/main/java/com/stripe/android/connect/analytics/ConnectAnalyticsEvent.kt

```diff
@@ -196,17 +196,13 @@ internal sealed class ConnectAnalyticsEvent(
      * Catch-all event for unexpected client-side errors.
      */
     data class ClientError(
-        val domain: String,
-        val code: Int,
-        val file: String,
-        val line: Int
+        val error: String,
+        val errorMessage: String? = null,
     ) : ConnectAnalyticsEvent(
         "client_error",
         mapOf(
-            "domain" to domain,
-            "code" to code.toString(),
-            "file" to file,
-            "line" to line.toString()
+            "error" to error,
+            "errorMessage" to errorMessage,
```

Comment:
Convention seems to be snake_case
```suggestion
            "error_message" to errorMessage,
```

Reply:
➕